draft-ietf-httpbis-incremental.md

```diff
@@ -193,6 +218,28 @@ Comments:
 : None
 {: spacing="compact"}
 
+An HTTP Proxy Error Type is registered in the HTTP Proxy Error Types registry as
+shown below:
+
+Name:
+: incremental_refused
+
+Description:
+: The HTTP message contained the Incremental HTTP header field, but the
+  intermediary refused to forward the message incrementally.
+
+Extra Parameters:
+: none
+
+Recommended HTTP Status Code:
+: 403 if refusing to serve an incremental request, or 504 if refusing an
+  incremental response.
```

Comment:
```suggestion
: 501
```

Reply:
Also I really think that if we offer the ability for the intermediary to warn the server about lack of incremental support, all of this becomes less of a problem since this case will then be marginal.